blogposts/2021/release-3.35.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Sourcegraph 3.35 release'
+publishDate: 2021-12-21T10:00-07:00
+description: 'Sourcegraph 3.35 introduces the ability to publish changesets to multiple branches of a repository with a single batch change, plus precise Code Intelligence for Java, Scala, and Kotlin.'
+tags: [blog, release]
+slug: 'release/3.35'
+published: false
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.35/sourcegraph-3-35-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.35/sourcegraph-3-35-release.png
+changelogItems:
+  - description: Individual batch changes can publish multiple changesets to the same repository by specifying multiple target branches using the `on.branches` attribute. This allows you to efficiently update long-lived branches for recent changes or upgraded dependencies.
+    url: https://docs.sourcegraph.com/batch_changes/references/batch_spec_yaml_reference#on-repository
+    category: Batch Changes
+  - description: The search bar is no longer auto-focused when navigating between files. This allows you to utilize the keyboard shortcut Cmd+LeftArrow (or Ctrl-LeftArrow) to go back to the browser's previous page instead of moving the cursor to the first position of the search bar.
```

Comment:
```suggestion
  - description: The search bar is no longer auto-focused when navigating between files. This allows you to use the keyboard shortcut Cmd+LeftArrow (or Ctrl-LeftArrow) to go back to the browser's previous page instead of moving the cursor to the first position of the search bar.
```